### PR TITLE
fix(playgrounds): stringify non-string types for playground output VSCODE-466

### PR DESCRIPTION
**Pull request by Anemy**
[VSCODE-466](https://jira.mongodb.org/browse/VSCODE-466)

Happens with both `console.log` and `print`.
Using the following example:
```
console.log('test');
console.log({a: 2});
console.log(['a', {a: 3}])
```
| before | after |
| - | - |
| Screenshot 2023-09-01 at 12 35 06 PM | Screenshot 2023-09-01 at 12 33 07 PM |

### CODE REVIEW

**Review by alenakhineika — APPROVED** [review]
👍 Yeah having a test would be nice!

File: src/editors/playgroundController.ts [hunk 1]
```diff
@@ -573,7 +573,11 @@ export default class PlaygroundController {
 
     if (evaluateResponse?.outputLines?.length) {
       for (const line of evaluateResponse.outputLines) {
-        this._outputChannel.appendLine(line.content);
+        this._outputChannel.appendLine(
+          typeof line.content === 'string'
+            ? line.content
+            : JSON.stringify(line.content)
```
Comment (addaleax) [inline src/editors/playgroundController.ts:579]: Is `JSON.stringify()` the right stringifier here? We’re showing this in the output to humans, maybe something like `util.inspect()` would make more sense, if a user really wants JSON output they can always use `print(EJSON.stringify(...))` (or [hopefully soon](https://jira.mongodb.org/browse/MONGOSH-1361) `printejson(...)`)?

Comment (alenakhineika) [reply src/editors/playgroundController.ts:579]: I like the idea of using util.inspect() here 👍

Comment (Anemy) [reply src/editors/playgroundController.ts:579]: Yes it definitely would, thanks for the suggestion.
Updated and added a test.

**Review by addaleax — APPROVED** [review]

**Pull request merged**